src/Entities/Entity.cpp

```diff
@@ -457,10 +457,15 @@ bool cEntity::DoTakeDamage(TakeDamageInfo & a_TDI)
 					case mtSilverfish:
 					{
 						a_TDI.RawDamage += static_cast<int>(ceil(2.5 * BaneOfArthropodsLevel));
-						// TODO: Add slowness effect
+						// The duration of the effect is a random value between 1 and 1.5 seconds at level I,
+						// increasing the max duration by 0.5 seconds each level, up to 3.5s with Bane of Arthropods V.
+						// Ref: https://minecraft.gamepedia.com/Enchanting#Bane_of_Arthropods
+						double MaxDuration = std::min(1 + BaneOfArthropodsLevel * 0.5, 3.5);
+						double Duration = GetRandomProvider().RandReal<double>(1, MaxDuration * 20);
+						Monster->AddEntityEffect(cEntityEffect::effSlowness, static_cast<int>(ceil(Duration)), 4);
```

Comment:
Perhaps the word bias wasn't strong enough. This is what I'm talking about:
![boa](https://user-images.githubusercontent.com/13238737/29495508-dfd94016-85b8-11e7-895a-136d325adb99.png)
Out of 1 million random numbers generated with this method, the minimum value was generated _0 times_.  There's no reason to use double and it comes with huge correctness issues.
> Say you have a problem, so you use floating-point.
Now you have 2.000001 problems.